autogpt/config/ai_config.py

```diff
@@ -46,71 +49,157 @@ def __init__(
             ai_role (str): The description of the AI's role.
             ai_goals (list): The list of objectives the AI is supposed to complete.
             api_budget (float): The maximum dollar value for API calls (0.0 means infinite)
+            plugins (list): The list of objectives the AI is supposed to complete.
         Returns:
             None
         """
         if ai_goals is None:
             ai_goals = []
+        if plugins is None:
+            plugins = []
         self.ai_name = ai_name
         self.ai_role = ai_role
         self.ai_goals = ai_goals
         self.api_budget = api_budget
+        self.plugins = plugins
```

Comment:
```suggestion
        self.plugins = plugins or []
```

This way you don't need lines 58-59